frontends/infinite-corridor/jest.config.ts

```diff
@@ -1,13 +1,12 @@
-import type { Config } from '@jest/types'
+import type { Config } from "@jest/types"
```

Comment:
This change, and all changes outside of `ol-template`, are because the list of files that is now being linted/formatted got bigger.